azure/client.go

```diff
@@ -108,85 +111,43 @@ func (c Client) Get(blobName string, snapshot time.Time) ([]byte, error) {
 	return data, nil
 }
 
-func (c Client) GetRange(blobName string, startRangeInBytes, endRangeInBytes uint64, snapshot time.Time) (io.ReadCloser, error) {
-	client, err := storage.NewClient(c.storageAccountName, c.storageAccountKey, c.baseURL, storage.DefaultAPIVersion, true)
-	if err != nil {
-		return nil, err
-	}
+// DownloadToFile download specified blobName to specified file
+func (c Client) DownloadBlobToFile(blobName string, file *os.File) error {
 
-	var snapshotPtr *time.Time
-	if !snapshot.IsZero() {
-		snapshotPtr = &snapshot
-	}
+	u, _ := url.Parse(fmt.Sprintf("https://%s.blob.core.windows.net/%s/%s", c.storageAccountName, c.container, blobName))
```

Comment:
Unfortunately, you can't assume `blob.core.windows.net`. This needs to be changed to reference the `c.baseURL` instead. Also preferably we wouldn't throw the error away and return the error if it is malformed.